components/combobox/combobox.jsx

```diff
@@ -182,7 +182,8 @@ const propTypes = {
 	 *
 	 * _Tested with snapshot testing._
 	 */
-	menuItem: PropTypes.func,
+	menuItem: PropTypes.func, // For backward compatibiity, 'menuItem' will be deprecated soon
```

Comment:
`menuItem` should be removed from `proptypes` object.

The doc website prop descriptions are generated from this code. `onRenderMenuItem` should be in the code right after 
```
*
 * _Tested with snapshot testing._
 */
```

Reply:
Thanks! Initially, I thought it would be required here to check the prop. Now, I got that, I will push the required changes shortly!